packages/core/src/browser/dialogs.ts

```diff
@@ -156,6 +168,8 @@ export abstract class AbstractDialog<T> extends BaseWidget {
 
         this.contentNode = document.createElement('div');
         this.contentNode.classList.add('dialogContent');
+        this.contentNode.setAttribute('style', `max-width: ${props.maxWidth} px`);
```

Comment:
@kaiyue0329 do you think we should limit the width on the content node or the entire dialog?
If we only limit the content, I think the props may need to be updated (or at least their documentation), since it's misleading.

Reply:
I think we should limit the width on the entire dialog, just so maxWidth is applied to the dialog title as well.

I updated the code.